source/hardware/controllers/denon_mc7000.rst

```diff
@@ -3,14 +3,18 @@ Denon MC7000
 
 The Denon MC7000 is a professional DJ controller which has got 4-channel capability and dual USB connections. These two USB audio interfaces enable two DJs to play together. This controller includes Denon’s high build quality and superior 24-bit audio reproduction, makes this suited to both mobile and club DJs.
 
+.. versionadded:: 2.2.4
+.. versionchanged:: 2.3.0
+   Fixed a bug for Vinylmode on Deck 4 and added new features, like library sorting, search through track with Jog Wheel, eject track from Deck, waveform zoom, Fixed Loops, changed Parameter and Censor/Reverse/Spinback button mapping. Added an experimental beat counter (slicer-like).
```

Comment:
```suggestion
   Fixed a vinyl mode bug on deck 4 and added new features, like library sorting, searching through track with jog wheel, ejecting track from deck, waveform zoom, fixed loops and an experimental beat counter (slicer-like) and improved parameter and censor/reverse/spinback button mappings. 
```

Reply:
Funny you are changing your own wording but I am okay with that too so just copy/paste it.... Cheers!

Reply:
Hehe, are you sure? "Vinylmode" without a space and with a larger "V" doesn't look like something I'd write. Or maybe I was on my phone and autocorrect kicked in 😁